docs/devguide.md

```diff
@@ -260,6 +260,21 @@ cluster you regularly use and are familiar with.  One of the choices you can
 make when deploying the catalog is whether to make the API server store its
 resources in an external etcd server, or in third party resources.
 
+If you have recently merged changes that haven't yet made it into a
+release, you probably want to deploy the canary images.
```

Comment:
You might add something like "Always use the canary images when testing local changes."